examples/tutorials/audio_data_augmentation_tutorial.py

```diff
@@ -10,6 +10,11 @@
 # with the following.
 # !pip install torchaudio
 
+from IPython.display import Audio, display
+import matplotlib.pyplot as plt
+import requests
+import os
+import math
```

Comment:
The imports are separated intentionally. Can we work around this?

Reply:
Sure. I will revert it to the original version.